src/views/EditDog.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Header from '../components/Header';
+import Form from '../components/Form';
+import { useState, useEffect } from 'react';
+import { getDogById } from '../services/dogData';
+import { updateDog } from '../services/dogData';
+
+export default function EditDog(props) {
+  const [loading, setLoading] = useState(true);
+  const [name, setName] = useState('');
+  const [age, setAge] = useState('');
+  const [bio, setBio] = useState('');
+  const [image, setImage] = useState('');
+  const [breed, setBreed] = useState('');
+  const [success, setSuccess] = useState('');
```

Comment:
`success` may not be the best name here as this state will contain either a success or error message